.github/workflows/mdbook_cicd.yml

```diff
@@ -0,0 +1,58 @@
+name: Spec MDBook CI
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2.3.1
+
+      - uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      - name: Use Node.js 🐰
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14.17.3'
+
+      - name: Update npm to v7 🔼
+        run: npm i -g npm
```

Comment:
Our package.json specifies `"engines": {"node": ">=16.14.0 <17"},`. I think we can move this to Node 16. Then we can remove the npm upgrade action. (16 defaults to npm v7+)